src/auto/application.rs

```diff
@@ -178,9 +178,6 @@ pub trait GtkApplicationExt: 'static {
     #[cfg(any(feature = "v3_24", feature = "dox"))]
     fn get_property_screensaver_active(&self) -> bool;
 
-    #[cfg(any(feature = "v3_24_8", feature = "dox"))]
-    fn connect_query_end<F: Fn(&Self) + 'static>(&self, f: F) -> SignalHandlerId;
```

Comment:
Why is this removed?

Reply:
I don't know. That might be the result of another change than mine.

Reply:
@antoyo  Seems you regenerated with old gir-files